app/models/course/condition/achievement.rb

```diff
@@ -20,11 +20,12 @@ def satisfied_by?(course_user)
 
   private
 
-  # Given an achievement, returns all its achievement conditions.
+  # Given a conditional object, returns all achievements that it requires.
   #
-  # @param [Course::Achievement] achievement The achievement to return achievement conditions for.
+  # @param [Object] conditional The object that is declared as acts_as_conditional and for which
```

Comment:
`acts_as_conditional` is defined under 
```
module Extensions::Conditional::ActiveRecord::Base
  module ClassMethods
    # Functions from conditional-and-condition framework.
    # Declare this function in the conditional model that requires conditions.
    def acts_as_conditional
        ...
    end
  end
end
```

Any class subclassing `ActiveRecord::Base` can add `acts_as_conditional`. What module should I specify? `ActiveRecord::Base`? And how should I specify it in the documentation part?